test/DynamoCoreWpfTests/DynamoConverterTest.cs

```diff
@@ -170,6 +170,16 @@ public void ConvertBetweenUnitsTestForForceReExecute()
             Thread.Sleep(500);
 
             AssertPreviewValue("45f1ee23-5d81-4233-975e-faf218203de5", 10000.0);
+
+            /*Convert from CubicMilliMeters to CubicMeters */
+            node1.Value = "1000000000";
+            converterNode.SelectedFromConversion = ConversionUnit.CubicMillimeter;
+            converterNode.SelectedToConversion = ConversionUnit.CubicMeters;
+            ViewModel.HomeSpace.Run();
+            Thread.Sleep(500);
```

Comment:
did you find this was required?

Reply:
I don't know whether this is required. May be I should remove this and run the self serve CI. This code is  3 year old now.

Reply:
I'm confused, this looks like new code you added?

Reply:
Nope..the other `thread.sleep` code. I removed this, and running self serve CI

Reply:
oh I see, that was following the pattern.